features/bootstrap/MfaContext.php

```diff
@@ -716,7 +675,7 @@ public function theUsersBrowserDoesNotSupportUf()
             'Update USER_AGENT_WITHOUT_WEBAUTHN_SUPPORT to a User Agent without WebAuthn support'
         );
         
-        $this->driver->getClient()->setServerParameter('HTTP_USER_AGENT', $userAgentWithoutWebAuthn);
+//        $this->driver->getClient()->setServerParameter('HTTP_USER_AGENT', $userAgentWithoutWebAuthn);
```

Comment:
This also looks like it would break some tests. Why comment it out? (Thanks for helping me understand.)